spec/mailers/user_mailer_spec.rb

```diff
@@ -11,6 +11,7 @@
     ActionMailer::Base.delivery_method = :test
     ActionMailer::Base.perform_deliveries = true
     ActionMailer::Base.deliveries = []
+    ActionMailer::Base.default_url_options[:host] = Rails.application.routes.default_url_options[:host]
```

Comment:
Yeah. But as far as I remember the problem was that in tests Spree config overrides `action_mailer.default_url_options` so that's why I had to override once again to make tests pass.